src/main/com/aca/homework/week12/params/UserCreateParams.java

```
@@ -0,0 +1,30 @@
+package com.aca.homework.week12.params;
+
+public class UserCreateParams {
+
+    private final String username;
+    private final String firstName;
+    private final String age;
+
+    public UserCreateParams(String username, String firstName, String age) {
+        this.username = setNotNullString(username);
```

this is a wrong approach. Please throw an exception instead